cluster/gce/gci/configure-helper.sh

```diff
@@ -1070,7 +1098,7 @@ function start-rescheduler {
 
 function reset-motd {
   # kubelet is installed both on the master and nodes, and the version is easy to parse (unlike kubectl)
-  local -r version="$(/usr/bin/kubelet --version=true | cut -f2 -d " ")"
+  local -r version="$("${KUBE_HOME}"/bin/kubelet --version=true | cut -f2 -d " ")"
```

Comment:
Does it make sense to also log the kubelet install location here? (i.e., whether its "${KUBE_HOME}"/bin/kubelet or /usr/bin/kubelet).
(again, only to avoid confusion).